app/routes/project/index.js

```diff
@@ -10,15 +10,13 @@ export default Route.extend({
   userSubscriptions: service(),
 
   model() {
-    let project = this.modelFor('project');
-
-    return RSVP.hash({
-      project,
-      subscription: this.get('userSubscriptions').fetchForProject(project)
+    return this.modelFor('project').reload().then((project) => {
```

Comment:
@begedin I think I agree with your logic here and would also like to go with @pixelhandler's approach [here](https://github.com/code-corps/code-corps-ember/pull/819#issuecomment-264545094).

Can we simplify to whatever you feel is the best simplification for now, and open an issue to add channels for project changes?